schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
-
+-- Create Database;
+CREATE DATABASE VetClinic;
+-- Create Table;
 CREATE TABLE animals (
-    name varchar(100)
-);
\ No newline at end of file
+    name varchar(100),
+    id integer,
+    date_of_birth date,
+    escape_attempts integer,
+    neutered: boolean,
```

Comment:
- [ ] It would be better to remove the colon next to neutered in order to create the table without any error 

```suggestion
    neutered boolean,
```